package.json

```diff
@@ -40,19 +41,19 @@
     "mochainon": "^2.0.0",
     "mockttp": "^0.9.1",
     "require-npm4-to-publish": "^1.0.0",
-    "resin-auth": "^2.0.0",
+    "balena-auth": "^3.0.0",
```

Comment:
These should be alphabetised in the way that `npm --save` would

Reply:
True, done